src/migrations/create-user.js

```diff
@@ -32,7 +41,7 @@ module.exports = {
     updatedAt: {
       allowNull: false,
       type: Sequelize.DATE
-    }
+    },
```

Comment:
Please rename this file back to 20190815191222-create-user.js for migration to work properly.